src/mount.coffee

```diff
@@ -290,7 +348,7 @@ class Mounter
     # Handle trace subprotocol
     switch command
       when 'start'
-        @tracer.attach @instance.network
+        @tracer.attach @instances[0].network
```

Comment:
This is broken. Should loop through the instances and add the network of each one